### PR TITLE
Fix a miscommit

### PR DESCRIPTION
**Pull request by lec-bit**
The last mistaken submission caused performance degradation and slow log printing.

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by lec-bit** [comment]
Multiple interactions take up a lot of performance, resulting in slow pod management, which may take more than ten seconds to respond.

File: pkg/controller/manage/kmesh_manage.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ var (
 )
 
 const (
-	DefaultInformerSyncPeriod = 2 * time.Second
+	DefaultInformerSyncPeriod = 30 * time.Second
```
Comment (LiZhenCheng9527) [inline pkg/controller/manage/kmesh_manage.go:57]: Is there no way to optimise it?

Comment (lec-bit) [reply pkg/controller/manage/kmesh_manage.go:57]: Synchronization here shouldnot be too frequent, and the interval should be as long as possible. In fact, if there are changes, they will be pushed over immediately, and there is no need to refresh them here regularly.

Comment (hzxuzhonghu) [reply pkg/controller/manage/kmesh_manage.go:57]: I would suggest removing resync https://github.com/kmesh-net/kmesh/issues/597

**Comment by hzxuzhonghu** [comment]
not needed anymore

**Pull request closed without merging**